components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -642,7 +644,27 @@ public enum ErrorMessages {
                 "attribute for organization with ID: %s."),
         ERROR_CODE_ERROR_LISTING_ORGANIZATIONS_DISCOVERY_ATTRIBUTES("65122", "Error while listing " +
                 "organizations discovery attributes.", "Server encountered an error while fetching discovery " +
-                "attributes for organizations.");
+                "attributes for organizations."),
+        ERROR_CODE_ERROR_CREATE_ORGANIZATION_USER_ASSOCIATION("65123", "Unable to create organization user " +
+                "association.", "Server encountered an error while creating organization user association " +
+                "for user %s."),
+        ERROR_CODE_ERROR_DELETE_ORGANIZATION_USER_ASSOCIATION_FOR_SHARED_USER("65124", "Unable to delete" +
+                " organization user association.", "Server encountered an error while deleting organization " +
+                "user association of the shared user %s."),
+        ERROR_CODE_ERROR_DELETE_ORGANIZATION_USER_ASSOCIATIONS("65125", "Unable to delete" +
+                " organization user associations of the user.", "Server encountered an error while deleting " +
+                "organization user associations of user %s."),
+        ERROR_CODE_ERROR_GET_ORGANIZATION_USER_ASSOCIATIONS("65126", "Unable to get the organization user " +
+                "associations.", "Server encountered an error while while fetching organization user " +
+                "associations for the user %s."),
+        ERROR_CODE_ERROR_GET_ORGANIZATION_USER_ASSOCIATION_FOR_USER_AT_SHARED_ORG("65127", "Unable to get the " +
+                "organization user association.", "Server encountered an error while fetching the organization " +
+                "user association for the user %s at shared organization %s."),
+        ERROR_CODE_ERROR_DELETE_SHARED_USER("65128", "Unable to remove the shared user.",
+                "Server encountered an error while deleting the shared user: %s from the shared organization %s"),
```

Comment:
```suggestion
                "Server encountered an error while deleting the shared user with ID: %s from the shared organization with ID: %s"),
```

Reply:
What is the user ID we will log here? Is that user's actual id or shared user reference?

Reply:
In this case, it is shared user reference. But there were some places where real user ID was logged in server exception is occurred. Those places were removed by this commit - https://github.com/wso2/identity-organization-management-core/commit/e8cddab79f478dcaa7abda1feb3783a0ec42790b